packages/web3-constants/evm/coingecko.json

```diff
@@ -34,7 +34,7 @@
         "BSCT": "",
         "Matic": "matic-network",
         "Mumbai": "",
-        "Arbitrum": "arbitrum-one",
+        "Arbitrum": "ethereum",
```

Comment:
Some token may exist on arb only.

Reply:
In the Arbitrum network, the arb token does not act as a native token for gas, but rather as an ethereum